sdk/mixedreality/mixed-reality-authentication/src/util/base64.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { isNode } from "@azure/core-http";
+import { isNode } from "@azure/core-util";
```

Comment:
we haven't released a GA version of core-util. Do we want to depend on it? For library code it is probably better to include a copy of `isNode`.  For tests (dev dependency) it is fine.

Reply:
That makes sense, I'll do the copy instead.